contracts/svm/testing/tests/submit_bid.rs

```diff
@@ -62,9 +63,10 @@ fn setup_bid() -> BidInfo {
     let deadline: i64 = 100_000_000_000;
     let ixs = [do_nothing_instruction(&searcher, permission_key, router)];
 
-    BidInfo {
+    BidSetupResult {
         svm,
         relayer_signer,
+        secondary_relayer_signer: setup_result.secondary_relayer_signer,
```

Comment:
it seems inconsistent with how the other fields from `setup_result` are passed around. 
`let secondary_relayer_signer = setup_result.secondary_relayer_signer`
you could also deconstruct `SetupResult`